.github/workflows/release.yml

```diff
@@ -0,0 +1,92 @@
+name: Create GitHub Release
+
+env:
+  RUNTIME_DEPENDENCIES: '[ libudev1, libevdev2, libglib2.0-0, libgee-0.8-2, upower ]'
+  UBUNTU_RELEASE: jammy
+  UBUNTU_PUBKEY: 871920D1991BC93C
+
+on:
+  push:
+  pull_request:
```

Comment:
Will this trigger when I create a release from GitHub's Releases tab? Because that's how I usually do it.

Reply:
> Runs your workflow when you push a commit *or tag*, or when you clone a repository.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push

Reply:
The process will reversed. You don't create a release manually anymore. You push a (version) tag, then the workflow creates a release and attaches the artifacts. It is possible to modify releases after their creation, so you can add a description if you like.

Reply:
Creating release from the interface creates a tag, so I'd expect files to be added to release after workflow finishes?

Reply:
Maybe, I don't know (yet). I think it should work as you expect, but it could be that the workflow fails because the release already exists. I'll try and report back.

Reply:
It works as you expect: you create a release in the UI using a new tag. The release is created immediately and contains only the sources. The workflow is starting. On success, the artifacts are added to the release.